contracts/token/ERC20/extensions/draft-ERC20Wrapper.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "../ERC20.sol";
+import "../utils/SafeERC20.sol";
+
+abstract contract ERC20Wrapper is ERC20 {
```

Comment:
Does this have to be abstract? It sounds useful on its own.

Reply:
For it to be non-abstract, the constructor needs to take a name and symbol and pass them through. I'm not a big fan of the idea. I'd prefer a preset, or support in the wizard.